src/python/WMCore/Configuration.py

```diff
@@ -357,6 +357,15 @@ def listSections_(self):
         comps = self._internal_settings
         return list(comps)
 
+    def getInternalName(self):
```

Comment:
Can you please make a simple unit test for this new method? Please keep it in a separate commit.